softwarereview_editor_management.pt.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+aliases: editorialmanagement.html
+---
+
+# Gerenciamento editorial {#editorialmanagement}
+
+```{block, type="summaryblock"}
+Orientação para gerenciar a equipe editorial.
+```
+
+## Recrutamento de novos editores {#recruiting-new-editors}
+
+O recrutamento de novos editores e a manutenção de um conselho editorial suficiente e equilibrado é responsabilidade da [Líder de revisão de software](https://ropensci.org/about/#team) com o apoio e a orientação do conselho editorial.
+
+Etapas:
+
+- Inicie um canal privado para discussão (para que você não tenha um histórico no canal de editores no qual futuros editores entrarão, o que pode ser incômodo).
+
+- Faça ping nos editores para garantir que eles recebam uma notificação, pois esse é um tópico importante.
+
+- Espere que a maioria dos editores se manifeste antes de convidar alguém. Dê a eles uma semana para responder.
+
+## Convidando um novo editor {#inviting-a-new-editor}
+
+- Os candidatos podem começar por ser [editores convidados](#guesteditor).
+  Ao convidá-los como editores convidados, convide-os da mesma forma que você convidaria um editor convidado por outros motivos.
```

Comment:
Mudar 'Ao convidá-los como editores convidados' para 'Ao convidá-los como editor convidado'.

Reply:
Suggestions included in the review.